src/System.Text.RegularExpressions/tests/GroupCollectionReadOnlyDictionaryTests.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Text.RegularExpressions.Tests
+{
+    public static class GroupCollectionReadOnlyDictionaryTests
+    {
+        [Fact]
+        public static void IReadOnlyDictionary_TryGetValueSuccess()
+        {
+            Regex regex = new Regex(@"(?<A1>a*)(?<A2>b*)(?<A3>c*)");
+            Match match = regex.Match("aaabbccccccccccaaaabc");
+
+            var groups = match.Groups;
```

Comment:
Nit: var => whatever type this is... same for the other occurrences of this in the other tests below